src/cephadm/cephadm

```diff
@@ -689,12 +689,12 @@ def dict_get(d: Dict, key: str, default: Any = None, require: bool = False) -> A
     """
     if require and key not in d.keys():
         raise Error('{} missing from dict'.format(key))
-    return d.get(key, default)
+    return d.get(key, default) # type: ignore
 
 ##################################
 
 
-def dict_get_join(d: Dict, key: str) -> Any:
+def dict_get_join(d: Dict, key: str) -> Any: # type: ignore
```

Comment:
which type error do you try to avoid here?

Reply:
see above; most of the errors being avoided on function declarations are due to pylance being really annoyed at the 'Any' keyword being returned.